src/engine/framework/System.cpp

```diff
@@ -124,7 +124,7 @@ static void CreateSingletonSocket()
 	addr.sun_family = AF_UNIX;
 	Q_strncpyz(addr.sun_path, singletonSocketPath.c_str(), sizeof(addr.sun_path));
 	if (bind(singletonSocket, reinterpret_cast<struct sockaddr*>(&addr), sizeof(addr)) == -1)
-		Sys::Error("Could not bind singleton socket: %s", strerror(errno));
+		Sys::Error("Could not bind singleton socket: %s, %s", strerror(errno), singletonSocketPath.c_str());
```

Comment:
The wording could be better, maybe `Could not bind singleton socket at <file>: <error>`